Backend/index.js

```diff
@@ -11,3 +18,11 @@ app.get("/hello", (req, res) => {
 app.listen(PORT, () => {
   console.log(`Listening on port ${3000}`);
 });
+
+mongoose
+  .connect(
+    "mongodb+srv://boorgusiddharth:siddharthjuly99@siddharth.fiuilki.mongodb.net/?retryWrites=true&w=majority&appName=Siddharth"
```

Comment:
Put db string and other important stuff like like API keys in a different folder because we do not want others to get access to this data.